async/kafka/kafka.go

```diff
@@ -112,14 +110,18 @@ func (f *Factory) Create() (async.Consumer, error) {
 	c := &consumer{
 		brokers:     f.brokers,
 		topic:       f.topic,
-		group:       f.group,
 		traceTag:    opentracing.Tag{Key: "group", Value: f.group},
```

Comment:
this applies only for consumer group but we add it to the trace anyway but it seems not to be used...